clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/AuthorizationSupport.groovy

```diff
@@ -53,6 +53,30 @@ class AuthorizationSupport {
     return true
   }
 
+  /**
+   *   Used for filtering result lists by searching for "account"-like properties.
+   */
+  boolean filterForAccounts(List items) {
+    if (!items) {
+      return true
+    }
+
+    Authentication auth = SecurityContextHolder.context.authentication;
+
+    new ArrayList<>(items).each { Object item ->
+      Map propertySource = item.properties
+      if (item instanceof Map) {
```

Comment:
The `items` parameter is a `List`, and it would be a very weird design for a REST method to return a list of lists without another kind of structure in between (say, a Map). In this case, comments can do nothing to correct bad design.